### DISCUSSION TITLE
is there a client sdk for this plugin api?

### DISCUSSION THREAD

**Discussion by rowthan** [coddingtonbear/obsidian-local-rest-api#91] [Q&A]
Has this project provided a NPM package for developer to call these local rest api ?

such as :

```
sdk.rest.search().then((res)=>{
 // get the result
})
```

**Comment by coddingtonbear** [comment]
There isn't, no, but maybe you could be the one to create such a project!

**Reply by rowthan** [reply]
If you are willing to accept it, I want to do it.

**Reply by coddingtonbear** [reply]
I'm not sure I'm up for maintaining that kind of thing as part of this project itself, unfortunately, but I'd be glad to help you out if you were to run into questions when creating a separate project for that and would add a link to your project from the project readme once you had a working implementation.
